Dockerfile-centos.template

```diff
@@ -43,17 +43,20 @@ EXPOSE 4466
 
 RUN yum update -y && yum clean all
 
-RUN useradd --system idsvr
+RUN groupadd --system --gid 10000 idsvr && useradd --system --no-create-home --uid 10001 --gid 10000 idsvr
 
 ENV IDSVR_HOME /opt/idsvr
 ENV JAVA_HOME $IDSVR_HOME/lib/java/jre
 ENV PATH $IDSVR_HOME/bin:$JAVA_HOME/bin:$PATH
+ENV LD_LIBRARY_PATH=/usr/local/lib64
```

Comment:
Is it empty by default, i.e. no need to concat with existing value?

Reply:
It is not necessary to set this actually, i removed it now. It was there from the previous tries i did where i copied the whole openssl build into `/usr/local/lib64` but i changed that after some testing last week and forgot to remove this.